flake.nix

```diff
@@ -204,7 +204,7 @@
     juno-src.url = "github:CosmosContracts/juno/v25.0.0";
     juno-src.flake = false;
 
-    osmosis-src.url = "git+https://github.com/osmosis-labs/osmosis?ref=refs/tags/v28.0.0";
+    osmosis-src.url = "github:osmosis-labs/osmosis/v30.0.1";
```

Comment:
should we use this again ? this pulls tarball from Github which may fail.

ref: #284

Reply:
Oh good point, I missed #284 and was confused by this change. Will update that

Reply:
Done in e097eedf3a6eec6d2258b2650520a14a83dcde13